app/assets/javascripts/admin/products.coffee

```diff
@@ -0,0 +1,3 @@
+# Place all the behaviors and hooks related to the matching controller here.
```

Comment:
You don't need to add files in your commits that you are not using. You can remove this file. Similar for "product.scss", "products_helper"